Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest AS builder
 
-ENV PINOT_VERSION=0.5.0
+ENV PINOT_VERSION=0.6.0
```

Comment:
Please update the service manager to match in `pinot-servicemanager/install.sh` and remove the warning on versions in `pinot-udf/README.md`

Reply:
Done. Fixed it.